### ISSUE TITLE
dr.round does not work for TensorXf

### ISSUE THREAD

**Issue by PidgeyBE** [mitsuba-renderer/drjit#57]
Hello!

I'm playing around with Dr Jit and noticed that `dr.round()` does not work for TensorXf objects... Is this expected?

Minimal reproducible example:
```
import drjit as dr
from drjit.cuda.ad import TensorXf, Array3f

array = Array3f([1.1, 2.1, 3.1])
tensor = TensorXf([1.1 ,2.2 ,3.3], shape=(3,))

print(dr.round(array))
print(dr.round(tensor))
```
=> The last line will raise an error:
```
IndexError: slice_tensor(): too many indices specified!
```

Thanks for taking the time to help me!
Pj

**Comment by Speierers** [comment]
Thanks for reporting this. I will fix it for the next release.

**Comment by PidgeyBE** [comment]
Thanks a lot!!

**Comment by Speierers** [comment]
Should be fixed here: [15d490c](https://github.com/mitsuba-renderer/drjit/commit/15d490c0f4da2ac9f5f56c249eb2bcb6e6e64da2)
